Kernel/Kernel.st

```diff
@@ -0,0 +1,23 @@
+"
+    Copyright (c) 2020 LabWare.
+    See (MIT) license in root directory.
+"
+
+Class {
+	#name : #Kernel,
+	#superclass : #Module,
+	#instVars : [
+		'entryPointCode',
+                'entryPointReceiver',
+		'nilObj',
+		'trueObj',
+		'falseObj',
+		'dlopenPtr',
+		'dlsymPtr'
```

Comment:
These two seem to deserve being in a separate place. We haven't started the "platform api" (neither the cross platform part of it), but it looks like it's an appropriate moment to start it. What about creating a `Library` class in `System/Posix`, so that we can have an analogue in `System/Windows`?